doc/release-notes.rst

```diff
@@ -9,17 +9,16 @@ Zephyr project is provided as source code and build scripts for different
 target architectures and configurations, and not as a binary image. Updated
 versions of the Zephyr project are released approximately every three-months.
 
-All Zephyr project source code is maintained in a `GitHub repository`_;
-you can either download source as a tar.gz file (see the bottom of the
-`GitHub tagged releases`_ page corresponding to each release), or use
-Git clone and checkout commands, such as:
+All Zephyr project source code is maintained in a `GitHub repository`_.
 
-.. code-block:: shell
-
-   git clone https://github.com/zephyrproject-rtos/zephyr
-   cd zephyr
-   git checkout tags/v1.13.0
+For Zephyr versions up to 1.13, you can either download source as a tar.gz file
+(see the bottom of the `GitHub tagged releases`_ page corresponding to each
+release), or clone the GitHub repository.
 
+With the introduction of the :ref:`west` tool after the release of Zephyr 1.13,
+it is no longer recommended to download or clone the source code manually.
+Instead we recommend to follow the instructions in :ref:`get_the_code` to do
+so with the help of West.
```

Comment:
your past self is asking you to not capitalize west ;)

Reply:
All addressed